.github/workflows/build_packages.yml

```diff
@@ -214,11 +214,12 @@ jobs:
           path: artifacts/
 
       # Create the release tag separately to add a message to it
+      # NOTE: As per the github API, we need to create the tag object first, and then the git reference
       - name: Create release tag
         uses: actions/github-script@v7
         with:
           script: |
-            github.rest.git.createTag({
+            const tagRequest = await github.rest.git.createTag({
```

Comment:
this will create create a tag for all the dev builds too?

Reply:
No, this whole job only runs if you specify that you want it to create a release in workflow inputs. The behaviour stays the same, only the release tag gains an annotation with the otel version.